Gemfile

```diff
@@ -4,4 +4,9 @@ group :development do
   gem "chefstyle", git: "https://github.com/chef/chefstyle.git"
 end
 
+# TODO: resolve this before merging this PR.
+# need to pull in master until commit https://github.com/chef/chef-zero/commit/c67095f4da311feb3d63cfd35cb25ee07e49f4b2
+# ships in a tagged chef release, and also chef-zero master is currently broken.
```

Comment:
We should just get this resolved before merging. We need to:

1) Fix bug introduced into chef-zero master.
2) Tag chef-zero 2.5.1 with a sha that includes the addition of the org scoped user key endpoint.

Then we can remove this tag.